website/content/docs/internals/plugins/index.mdx

```diff
@@ -0,0 +1,39 @@
+---
+layout: docs
+page_title: Plugin System
+description: Learn about Vault's plugin system.
+---
+
+
+# Plugin System
+
+All Vault auth and secret backends are considered plugins. This simple concept
+allows both built-in and external plugins to be treated like Legos. Any plugin
+can exist at multiple different locations. Different versions of a plugin may
+be at each one, with each version differing from Vault's version.
```

Comment:
Suggestion: **Different versions of a plugin may be at each one**. "One" is considered an antecedent in this sentence and to provide clarity, we may want to replace 'one' with 'location' instead.